src/inference/src/system_conf.cpp

```diff
@@ -141,7 +155,24 @@ bool with_cpu_x86_avx512_core_amx_fp16() {
 bool with_cpu_x86_avx512_core_amx() {
     return false;
 }
-
+bool with_cpu_neon_fp16() {
+#    if !defined(_WIN64) && !defined(BARE_METAL) && !defined(__APPLE__) && !defined(__OpenBSD__) && \
```

Comment:
We usually don't add spaces after `#` symbol. Could you please change the formatiing? I think it will help from code readability standpoint.